### DISCUSSION TITLE
Bug: A user with an hobby plan can access 90 days web analytics

### DISCUSSION THREAD

**Discussion by ferrariofilippo** [vercel/community#2495] [Bug Report]
According to your pricing section, a user with an `Hobby Plan` should be able to see analytics for at most the past 30 days.
Indeed the drowdown in the analyitics page lets you select at most `30 days`.
The issue is that if you change the link manually (setting `/analytics?period=90d`), the user will be able to see data for the past 90 days.
![image](https://github.com/vercel/community/assets/102259289/d40a9dd3-e646-483b-ae6a-8be86b8fc203)
![image](https://github.com/vercel/community/assets/102259289/77128ec9-d0b7-4dfb-a65b-91589a2cabd3)

**Comment by amyegan** [comment]
Thanks for reporting this! I shared it with the analytics team 🙇

**Comment by ferrariofilippo** [comment]
It seems this is fixed
